docs/feature_flag_in_process_configuration.md

```diff
@@ -0,0 +1,151 @@
+# Feature Flag In-process Configuration
+
+The `FeatureFlagInProcessConfiguration` is a custom resource used to set up the
+[configuration options](https://flagd.dev/providers/nodejs/?h=flagd_host#available-configuration-options)
+for applications using `OpenFeature operator` with in-process evaluation mode enabled.
+
+Below you can see a minimal example of `FeatureFlagInProcessConfiguration` resource
+
+```yaml
+apiVersion: core.openfeature.dev/v1beta1
+kind: FeatureFlagInProcessConfiguration
+metadata:
+  labels:
+  name: featureflaginprocessconfiguration-sample
+spec:
+  port: 2424
```

Comment:
```suggestion
  port: 8015
```

This is to avoid any copy paste issue ;)

Reply:
This is on purpose to showcase that the port is being set and the value overrides the default value (8015)

Reply:
Yeah, that's fair argument :)